.github/workflows/integrationTest.yml

```diff
@@ -806,9 +808,86 @@ jobs:
           max_attempts: 3
           timeout_minutes: 8
           retry_wait_seconds: 5
-          command: cd terraform/ecs/linux && terraform destroy --auto-approve
+          command: cd terraform/ecs_fargate/linux && terraform destroy --auto-approve
+
+  StressTrackingTest:
+    name: "StressTrackingTest"
+    needs: [MakeBinary, EC2LinuxIntegrationTest, ECSEC2IntegrationTest, ECSFargateIntegrationTest, GenerateTestMatrix]
```

Comment:
Even though `StressTrackingTest` can run parallel with `EC2LinuxIntegrationTest`, `ECSEC2IntegrationTest`, and `ECSFargateIntegrationTest`. However, `StressTrackingTest` can claim resources and is not prioritized more than feature